packages/nhost-js/src/core/nhost-client.ts

```diff
@@ -39,8 +41,12 @@ export class NhostClient {
     start = true
   }: NhostClientConstructorParams) {
     if (!backendUrl) throw new Error('Please specify a `backendUrl`. Docs: [todo]!')
+
+    const subdomain = new URL(backendUrl).host.split('.')[0]
```

Comment:
Might cause problems if people use http://localhost:1337 as backendUrl and adds a region to the constructor.

Reply:
this was removed because we decided to also have `subdomain` as an option.